include/boost/geometry/util/range.hpp

```diff
@@ -116,10 +116,10 @@ front(Range const& rng)
 
 /*!
 \brief Short utility to conveniently return the front element of a Range.
-\ingroup utility
-*/
+ \ingroup utility
+ */
```

Comment:
I have done a `push --force` so that the PR remains a single simple commit (not three commits with space removed). It's bad practice, but I don't think anyone has pulled my branch in the latest 5 minutes... so it should be armless.